chainer_compiler/chainer_compiler.py

```diff
@@ -182,65 +183,51 @@ def backward(self, indexes, flat_gys):
         return gxs
 
 
-def _run_translator(translator, mc, inputs):
+def export(model, inputs, filename=None, translator='onnx_chainer'):
     if translator == 'ch2o':
         from chainer_compiler import ch2o
-        xmodel = ch2o.compile_model(mc, inputs)
-        f = tempfile.NamedTemporaryFile(delete=False)
+        xmodel = ch2o.compile_model(model, inputs)
+        if filename is None:
```

Comment:
Could you factor out line 190-193? Just moving 190-193 and 199-202 to the beginning of this function would be enough

Reply:
Sure, I'll do in future PR.